homeassistant/components/trafikverket_camera/entity.py

```diff
@@ -0,0 +1,56 @@
+"""Base entity for Trafikverket Camera."""
+from __future__ import annotations
+
+from homeassistant.core import callback
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import TVDataUpdateCoordinator
+
+
+class TrafikverketCameraEntity(CoordinatorEntity[TVDataUpdateCoordinator]):
+    """Base entity for Trafikverket Camera."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: TVDataUpdateCoordinator,
+        entry_id: str,
+    ) -> None:
+        """Initiate Trafikverket Camera Sensor."""
+        super().__init__(coordinator)
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, entry_id)},
+            entry_type=DeviceEntryType.SERVICE,
+            manufacturer="Trafikverket",
+            model="v1.0",
```

Comment:
Did we loose the name?

Reply:
Yes and no, if a device has no name it uses the config entry title, which we already did :)

Reply:
I suspected so but wasn't pure